README.md

```diff
@@ -7,12 +7,52 @@
 
 ## Sessionless
 
-*Sessionless* is an attempt to make authentication handling easier for developers and users, while also enabling interesting behavior not possible with traditional sessions.
-No emails, no passwords, and no sessions, just establish auth and get going.
-
 Sessionless is an authentication protocol, which can also be referred to as the SessionLess Authentication Protocol (SLAP).
+It is meant to be a (the following aren't links, they're on-hover texts) [general purpose][ht1] [transport-agnostic][ht2] [multi-language][ht3] [multi-platform][ht4] authentication protocol with no [shared-secret][ht5] or [personally identifying information][ht6] shared between participants. 
+
+### Getting Started
+
+You will want to checkout the language, and if necessary the platform, you're interested in in the repo for installation steps.
+In general, everything is available in the standard package manager system of the given language (i.e. maven for java, and pip for python).
+
+#### Methods
+
+There are four methods that each implementation has.
+In addition there are two optional method based on how the platform stores sensitive data.
+For the most part these optional methods are only available in client-side platforms like mobile, and consoles.
+
+The methods are as follows (see [API] for more details):
+
+**Client**
+
+* `generateKeys` - generates a private/public key pair on the secp256k1 elliptical curve
+* `sign` - creates a signature of a message string with a private key
+
+**Server**
 
-[Here is a simple video of what Sessionless can do.]
+* `verifySignature` - verifies a signature with a public key
+* `generateUUID` - generates a uuidv4 uuid
+
+**Optional**
+
+* `getKeys` - on platforms where secure storage is well-defined we've implemented getKeys within Sessionless rather than having it passed into generateKeys.
+* `associate` - convenience method for verifying two signatures to associate a public key with another
+
+A typical registration is for a client to `generateKeys`, and then use those keys to `sign` a message, and send to the server.
+The server can then `verifySignature` on the payload received from the client, and grant a uuid with `generateUUID`.
+
+#### Building with Sessionless
+
+Like https (I use the comparison simply for familiarity, and not try to say Sessionless is on the same tier), Sessionless is a protocol that you likely won't just use directly.
+Instead it's meant to be the protocol layer of an authentication _implementation_.
+One example implementation can be found [here].
+More will come, and if you make one feel free to make a pr to link it here.
+
+#### Want to learn more?
+
+There are links to more reading below, and some videos coming soon.
```

Comment:
I think it could be a good idea to add the links here instead of telling the reader to find them himself, it might improve the UX (idk if that matters here lol)

Reply:
good call